test-utils/assert/README.md

```diff
@@ -0,0 +1,84 @@
+ 
+```groovy
+complete(task(instance), withVariables("approved", true));
+```
+
+## Compatibility
+
+Camunda Platform Assert works with the corresponding version of Camunda Platform (i.e., Camunda Platform Assert 7.17.0 is compatible to Camunda Platform 7.17.0). The compatibility between earlier versions are as shown [in the documentation](https://docs.camunda.org/manual/latest/user-guide/testing/#assertions-version-compatibility).
+Camunda Platform Assert is works with multiple Java versions (1.8+). All of this is continuously verified by executing around 500 test cases. 
+
+## Get started
+
+1. Add a maven test dependency to your project:
+
+```xml  
+<dependency>
+    <groupId>org.camunda.bpm</groupId>
+    <artifactId>camunda-bpm-assert</artifactId>
+    <version>7.17.0</version>
+    <scope>test</scope>
+</dependency>
+```
+
+Additionally, [AssertJ](https://assertj.github.io/doc/) needs to be provided as a dependency with a version that is compatible with the one documented in the [compatibility matrix](https://docs.camunda.org/manual/latest/user-guide/testing/#assertions-version-compatibility).
+
+Please note that if you use [Spring Boot](https://spring.io/projects/spring-boot) or the [Camunda Spring Boot Starter](https://docs.camunda.org/manual/latest/user-guide/spring-boot-integration/) in your project, AssertJ is already included in your project's setup.
+
+2. Add a static import to your test class
+
+Create your test case just as described in the [Camunda Platform Testing Guide](https://docs.camunda.org/manual/latest/user-guide/testing/) and add Camunda Platform Assert by statically importing it in your test class:
+
+```groovy  
+import static org.camunda.bpm.engine.test.assertions.ProcessEngineTests.*;
+```
+
+3. Start using the assertions in your test methods
+
+You now have access to all the Camunda Platform assertions. Assuming you want to assert that your process instance is actually started, waiting at a specific user task and that task should yet be unassigned, but waiting to be assigned to a user of a specific group, just write:
+
+```groovy
+assertThat(processInstance).isStarted()
+  .task().hasDefinitionKey("edit")
+    .hasCandidateGroup("human-resources")
+    .isNotAssigned();
+```
+
+In case you want to combine Camunda Platform Assert with the assertions provided by AssertJ, your imports should look like this:
+```groovy  
+import static org.assertj.core.api.Assertions.*;
+import static org.camunda.bpm.engine.test.assertions.ProcessEngineTests.*;
+```
+
+Find a more detailed description of the assertions and helper methods available in the Camunda Platform Assert [user guide](https://github.com/camunda/camunda-bpm-assert/blob/master/docs/README.md).
```

Comment:
```suggestion
```

Will these docs eventually be moved to docs.camunda.org? I think we shouldn't link to a to be unmaintained repo.